apps/docs/src/guide/wallets/connectors.md

```diff
@@ -4,6 +4,422 @@
+
+It will return a promise that will resolve to `true` if the ABI information is available, otherwise `false`.
+
+<<< @/../../../packages/account/src/connectors/fuel-connector.ts#fuel-connector-method-hasABI{ts:line-numbers}
+
+## Fuel SDK
+
+The `Fuel` manager provides the interface for interaction with the TS SDK and the wider Fuel ecosystem.
+
+It can be instantiated as follows:
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-instantiation-1{ts:line-numbers}
+
+### Options
+
+There are a number of options that can be passed to the `Fuel` manager:
+
+- [`connectors`](#connectors)
+- [`storage`](#storage)
+- [`targetObject`](#targetObject)
+
+#### `connectors`
+
+The `connectors` option is used to provide a list of connectors that the `Fuel` manager can interact with. The manager will interact with the connectors that will in turn handle the communication with the respective wallet. You can find a list of all the connectors in our [`FuelLabs/fuel-connectors`](https://github.com/FuelLabs/fuel-connectors). Below you will find a snippet of using the current default connectors that are provided by the `fuel-connectors` package.
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-options-connectors{ts:line-numbers}
+
+#### `storage`
+
+The `storage` is used internally to store the current connector state. This can be overriden as follows, by passing an instance that extends the `StorageAbstract` class.
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-options-storage-memory{ts:line-numbers}
+
+The default behavior will use `LocalStorage` if the `window` is available:
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-options-storage-local{ts:line-numbers}
+
+#### `targetObject`
+
+The `targetObject` is used to provide a target for the `Fuel` manager to interact with. Used for registering events and can be overridden as follows:
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-options-target-object{ts:line-numbers}
+
+### Methods
+
+The `Fuel` manager provides a number of methods to interact with the SDK:
+
+#### All methods from connectors
+
+The `Fuel` manager provides all the [methods](#methods) that are available from the connectors that are currently connected. This means that you can interact with the current connector as if you were interacting with the `Fuel` manager directly.
+
+If there is no current connector available or connected, it will throw an error.
+
+#### `connectors`
+
+The `connectors` method is used to get the current list of _installed_ and _connected_ connectors that are currently available.
+
+<<< @/../../../packages/account/src/connectors/fuel.ts#fuel-sdk-method-connectors{ts:line-numbers}
+
+#### `getConnector`
+
+The `getConnector` method is used to resolve a connector by its name. This is useful for getting a specific connector that you want to interact with. If the connector is not found, it will return `null`.
```

Comment:
```suggestion
The `getConnector` method resolves a connector by its name. This is useful for finding a specific connector with which to interact. If the connector is not found, it will return `null`.
```

Reply:
09e0871ef593c3cdbf7aa799b437996ef2232e90